### PR TITLE
App/readme

### PR DESCRIPTION
**Pull request by JuanByte**
## :loudspeaker: Type of change

:white_small_square: Bugfix
:white_small_square: New feature
:white_small_square: Enhancement
:black_small_square: Refactoring
:white_small_square: Maintenance

## :scroll: Description and Motivation

Add instructions to compile the app in README and delete duplicated Models files 

## :pencil: Checklist

:white_small_square: Reviewed submitted code
:white_small_square: Added tests to verify changes
:white_small_square: Tracking approval, if applicable
:white_small_square: Verified UI on multiple screen sizes, if applicable
:white_small_square: Verified on lower API versions, if applicable
:white_small_square: Verified on bad connection, if applicable

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,47 @@
...
+
+* The MRPA show a list of Roberts photos sort by the sol(Martian rotation or day) on which they were taken displaying the image nicely to the user.
+* After an image is clicked the app opens its full side and its descriptions allowing the user to share it as well. 
+
+Getting Started
+---------------
+
+To be able to compile the app follow the instructions below:
+
+1. Download and run the app.
```
Comment (javierjmc) [inline README.md:18]: You can't run the app if you don't do the following steps :D

File: README.md [hunk 2]
```diff
@@ -1,10 +1,47 @@
...
+2. Get Nasa API service key from https://api.nasa.gov/. In the NASA API website head to the "Generate API key" section and fill out the    form. You should receive an email with your personal    key after sing up. 
+3. Once you have the key you need to define it in build.gradle. So head to the following path according to your OS. 
+   
+   Windows: C:\Users\<Your Username>\.gradle
+   Mac: /Users/<Your Username>/.gradle
+   Linux: /home/<Your Username>/.gradle
+ 
+ 4. You should fine a file named gradle.properties if is not there you must create it. Open gradle.properties and write the following       lines:
```
Comment (javierjmc) [inline README.md:26]: `fine` -> `find`

File: README.md (hunk 1 above)
Comment (JuanByte) [reply README.md:18]: hhahaha yeah right haha!!

**Comment by JuanByte** [comment]
I'll make the corrections and merge then. :)

**Pull request merged**